src/main/resources/edu/cornell/kfs/concur/cu-spring-concur.xml

```diff
@@ -391,4 +391,26 @@
         <property name="titleKey" value="message.concur.requestExtract.title.createPdpFeedFile"/>
 	</bean>
 
+    <!-- Standard Accounting Extract Batch Report -->
+    <bean id="concurStandardAccountingExtractReportService" parent="concurStandardAccountingExtractReportService-parentBean" />
+    <bean id="concurStandardAccountingExtractReportService-parentBean" class="edu.cornell.kfs.concur.batch.service.impl.ConcurStandardAccountingExtractReportServiceImpl" abstract="true">
+        <property name="reportWriterService" ref="concurReportWriterService" />
+        <property name="fileNamePrefixFirstPart" value="SAE_summary_for__"/>
+        <property name="fileNamePrefixSecondPart" value="__"/>
+        <property name="reportTitle" value="Summary of Concur Standard Accounting Extract to KFS PDP and KFS Collector Files"/>
+        <property name="reportConcurFileNameLabel" value="Report is for Concur Standard Accounting Extract File :          "/>
+        <property name="reimbursementsInExpenseReportLabel" value="Reimbursements In Expense Report :"/>
+        <property name="cashAdvancesRelatedToExpenseReportsLabel" value="Cash Advances Related to Expense Report :"/>
+        <property name="expensesPaidOnCorporateCordLabel" value="Expenses Paid on Corporate Card :"/>
+        <property name="transactionsByPassedLabel" value="By Passed Expense Type XXXX Transactions :"/>
```

Comment:
It would be a little clearer to use one word for "bypass" instead of two, and elsewhere in the code as needed.